.github/workflows/server-CI.yaml

```diff
@@ -0,0 +1,21 @@
+name: Server CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: server
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16'
+      - name: Install deps
+        run: yarn
```

Comment:
Do we need to yarn build?

Reply:
Nope. You can check the CI on my fork at: https://github.com/jazcarate/Drift/runs/5804912391?check_suite_focus=true